api/src/main/java/org/motechproject/nms/api/web/KilkariController.java

```diff
@@ -101,4 +104,73 @@ public void saveInboxCallDetails(@RequestBody InboxCallDetailsRequest request) {
         //todo: the work...
     }
 
+    /**
+     * 4.2.3
+     * Create Subscription
+     *
+     * IVR shall invoke this API to create a new Kilkari subscription
+     *
+     */
+    @RequestMapping(value = "/subscription",
+            method = RequestMethod.POST,
```

Comment:
ok, ignore the next comment that should be coming through the intertubes
right now then.

On Wed, Apr 22, 2015 at 4:40 PM, Lauren Lavoie <notifications@github.com>
wrote:

> In api/src/main/java/org/motechproject/nms/api/web/KilkariController.java
> <https://github.com/koshalt/mim/pull/73#discussion_r28925293>:
>
> > @@ -101,4 +104,73 @@ public void saveInboxCallDetails(@RequestBody InboxCallDetailsRequest request) {
> >          //todo: the work...
> >      }
> >
> > +    /**
> > +     * 4.2.3
> > +     * Create Subscription
> > +     *
> > +     * IVR shall invoke this API to create a new Kilkari subscription
> > +     *
> > +     */
> > +    @RequestMapping(value = "/subscription",
> > +            method = RequestMethod.POST,
>
> It is. The kilkari part is mapped at the class level.
> -----Original Message----- From: "Rob LaRubbio" <notifications@github.com>
> Sent: ‎4/‎22/‎2015 4:38 PM To: "koshalt/mim" <mim@noreply.github.com> Cc:
> "Lauren Lavoie" <llavoie@grameenfoundation.org> Subject: Re: [mim]
> Kilkari CreateSubscription and DeactivateSubscription (#73
> <https://github.com/koshalt/mim/pull/73>) In
> api/src/main/java/org/motechproject/nms/api/web/KilkariController.java:
>  @@ -101,4 +104,73 @@ public void saveInboxCallDetails(@RequestBody
> <https://github.com/RequestBody> InboxCallDetailsRequest request) {
> //todo: the work... } + /** + * 4.2.3 + * Create Subscription + * + * IVR
> shall invoke this API to create a new Kilkari subscription + * + */ +
> @RequestMapping(value = "/subscription", + method = RequestMethod.POST,
> Should the url be /kilkari/subscription? — Reply to this email directly or
> view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/koshalt/mim/pull/73/files#r28925293>.
>